tests/vcr_python_wrapper.py

```diff
@@ -9,6 +9,9 @@
 except ImportError:
     from urllib.parse import urlparse, urlunparse
 
+FILTER_REQUEST_HEADERS = ['Cookie']
+FILTER_RESPONSE_HEADERS = ['Apipie-Checksum', 'Date', 'ETag', 'Server', 'Set-Cookie', 'Via', 'X-Powered-By', 'X-Request-Id', 'X-Runtime']
```

Comment:
what about removing everything starting with "X-"?

Reply:
I thought about that, but that would require a different filter approach (loop over existing headers and drop the ones with X-) so I didn't for now.

Reply:
Or did you mean add all the available X headers to that list?

Reply:
By nature of the X-headers, that would be impossible.
I thought about `if header in FILTER or header.startswith("X-"):`

Reply:
Yeah I meant all currently used ;)

Reply:
Anyways, let's merge this and think about more filtering later ;)